common/common.h

```diff
@@ -153,6 +153,9 @@ struct gpt_params {
     bool prompt_cache_ro   = false; // open the prompt cache read-only and do not update it
 
     bool embedding         = false; // get only sentence embedding
+    int32_t embd_normalize = 2;     // normalisation for embendings (-1=none, 0=max absolute, 1=taxicab, 2=euclidean, >2=p-norm)
+    std::string embd_out   = "";    // empty = default, "array" = [] or [[],[]...], "json" = openai style, "json+" = same "json" + cosine similarity matrix
+    std::string embd_sep   = "\n";  // separator of embendings
```

Comment:
These group of parameters should be moved below in their own section, similar to what we do for other examples:

```c++
    // embedding
    int32_t embd_normalize = 2;     // normalisation for embedings (-1=none, 0=max absolute, 1=taxicab, 2=euclidean, >2=p-norm)

    std::string embd_out   = "";    // empty = default, "array" = [] or [[],[]...], "json" = openai style, "json+" = same "json" + cosine similarity matrix
    std::string embd_sep   = "\n";  // separator of embedings
```